src/utils/getAssetsFilters.jsx

```diff
@@ -0,0 +1,23 @@
+import ASSET_TYPES from '../data/constants/assetTypeFilters';
+
+export const getSelectedFilters = (filters) => {
+  if (filters === null || filters === undefined) {
+    return {};
+  }
+  return Object.keys(filters)
+    .filter(type => filters[type] === true);
+};
+
+export const hasSelectedFilters = filters => (
+  getSelectedFilters(filters).length > 0
+);
+
+export const getDefaultFilterState = () => (
```

Comment:
This returns the default initial filter state for the filter reducer. It is an object, where the keys are all the asset filters and the values are all false.